core/arch/arm/include/tee/entry.h

```diff
@@ -25,8 +25,8 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef TEE_ENTRY_H
-#define TEE_ENTRY_H
+#ifndef ENTRY_H
```

Comment:
Yes, I guess it's partially because we haven't enforced this much, but also because we've renamed many files.